### ISSUE TITLE
Issue when install sqlalchemy-utils 0.37.0 with pipenv

### ISSUE THREAD

**Issue by sbrunner** [kvesteri/sqlalchemy-utils#517]
I get
```
[pipenv.exceptions.InstallError]: Collecting sqlalchemy-utils==0.37.0
[pipenv.exceptions.InstallError]:   Using cached SQLAlchemy_Utils-0.37.0-py3-none-any.whl (99 kB)
[pipenv.exceptions.InstallError]: ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
[pipenv.exceptions.InstallError]:     sqlalchemy-utils==0.37.0 from https://files.pythonhosted.org/packages/7b/8a/09db63503bd95f334f51bcc07bec8c7fec818fcd753ea6d6e0288306ef45/SQLAlchemy_Utils-0.37.0-py3-none-any.whl#sha256=650534006910247670bd0ad0ad457944d6fc967f574165151f946b14d1ee823b (from -r /tmp/pipenv-urf0do_v-requirements/pipenv-shfzalww-requirement.txt (line 1)):
[pipenv.exceptions.InstallError]:         Expected sha256 c7bec2c982b31ec6133ba519f73f07653bbb7e7b3c23836bb8d9133045386b68
[pipenv.exceptions.InstallError]:              Got        650534006910247670bd0ad0ad457944d6fc967f574165151f946b14d1ee823b
ERROR: Couldn't install package: sqlalchemy-utils
```

**Comment by realerikrani** [comment]
I get this error too. What could cause it?

**Comment by sbrunner** [comment]
Thanks, but create a new release will be a better option :-)

**Comment by kvesteri** [comment]
Released new version 0.37.1
